public/js/service-worker-registration.js

```diff
@@ -0,0 +1,11 @@
+// register service worker
+
+if ('serviceWorker' in navigator) {
+    navigator.serviceWorker.register('/service-worker.js')
+        .then(function (registration) {
+            // console.log('Service worker registration successful, scope is:', registration.scope);
```

Comment:
Having this enabled in the logs in production is fine from my perspective.

```suggestion
            console.info('Service worker registration successful, scope is:', registration.scope);
```

It would be even better to have it in its own service since it could then use the `Logger` service but it should be fine for now.

Reply:
I went with your suggestion and enabled the `console.info` statement.